snuba/web/db_query.py

```diff
@@ -783,6 +783,12 @@ def db_query(
                 query_id=query_id,
                 result_or_error=QueryResultOrError(query_result=result, error=error),
             )
+        if stats.get("cache_hit"):
+            metrics.increment("cache_hit", tags={"dataset": dataset_name})
+        elif stats.get("cache_wait"):
```

Comment:
I don't think is correct, the code uses `is_duplicate`: https://github.com/getsentry/snuba/blob/master/snuba/web/db_query.py#L398

Reply:
@evanh Thanks for catching this error. I will update